docker/backend.Dockerfile

```diff
@@ -1,8 +1,9 @@
 ##################################################################################
 # BRANDED ########################################################################
 ##################################################################################
-FROM ocelotsocialnetwork/backend:latest as branded
+ARG APP_IMAGE=ocelotsocialnetwork/backend:latest
+FROM $APP_IMAGE as branded
 
-# Copy public constants to the docker image branding it
+# Copy public constants to the docker image and branding it
```

Comment:
```suggestion
# Copy public constants to the docker image branding it
```

Reply:
`... to the docker image branding it`
'... in das docker image um es zu branden`
or
`... to the docker image and brand it`
`... in das docker image und brande es`

I use the first because it describes that when you copy stuff like this to the docker image it becomes branded that way. The second sentence indicates that the copying and the branding are two separate processes which are done independently.

Reply:
Ah, okay 👍🏼
I’ll do later

Reply:
I would suggest:
`kopieren Sie öffentliche Konstanten in das Docker-Image, um es zu brandmarken` (brandmarken is the closest in german)
`copy public constants into the Docker image to brand it` (says Deepl)